.github/workflows/build.yml

```diff
@@ -76,3 +81,157 @@ jobs:
         run: meson build && ninja -C build
       - name: Install jack-example-tools
         run: ninja -C build install
+  build_pawpaw_macos_intel:
```

Comment:
Why do we need the macOS builds though?

Reply:
to test building against the official jack2 binaries.
the way homebrew builds things does not match jack2 releases. for one they do not support universal binaries.